src/test/java/uk/gov/pay/directdebit/payments/fixtures/PayersFixture.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.pay.directdebit.payments.fixtures;
```

Comment:
There is already https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/test/java/uk/gov/pay/directdebit/payers/fixtures/PayerFixture.java, is this a duplicate?